content/docs/command-reference/plot/diff.md

```diff
@@ -105,8 +105,9 @@ file:///Users/dmitry/src/plot/logs.csv.html
 
 ![](/img/plot_diff.svg)
 
-The predefined confusion matrix template shows how continuous metrics difference
-can be faceted by separate plots:
+The predefined confusion matrix (`.dvc/plot/confusion_matrix.json`)
+[template](/doc/command-reference/plot#plot-templates) shows how metric
+differences can be faceted by separate plots:
```

Comment:
Hi @dmpetrov can you explain what you meant here? I don't really understand the idea. What does faceted mean in this context? Thanks!